jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/RuntimeDataServiceImpl.java

```diff
@@ -700,6 +700,7 @@ public UserTaskInstanceDesc getTaskById(Long taskId) {
 		Map<String, Object> params = new HashMap<String, Object>();
         params.put("userId", userId);
         params.put("status", allActiveStatus);
+        params.put("groupIds", identityProvider.getRoles());
```

Comment:
roles should never be null as you need to be at least in one role to be able to use the system